### PR TITLE
Sidebar Width change

### PR DESCRIPTION
**Pull request by Jagdish-J-P**
| Question                       | Answer
| -----------------------    | -----------------------
| Issue or Enhancement  |  Enhancement
| License                          | MIT

#### What's in this PR?

- Added custom option to change sidebar width.

edit config/adminlte.php and modify sidebar_width to change width of sidebar.

```
'sidebar_width' => '250px',
```

#### Checklist

- [x] I tested these changes.

### CODE REVIEW

**Comment by Shidersz** [comment]
@Jagdish-J-P Are you sure this won't conflict with the underlying `AdminLTE` package?
@REJack can you give your opinion about this? I'm not sure if it's good to mess with the `AdminLTE` styles.

**Comment by Jagdish-J-P** [comment]
I have just overridden AdminLTE's sidebar width property. It won't affect any other thing. Or I can add another configuration to enable custom width so if it affects, user can disable custom width.

**Comment by Shidersz** [comment]
@Jagdish-J-P The `AdminLTE` package uses `$sidebar-width` variable of `build/scss/_variables.scss` file to setup the sidebar width. If you use `grep` (or similar tool) to search inside the `AdminLTE` package you can see this variable is used on a lot of places (see below):

### Result of `grep -r "sidebar-width" build/` inside `AdminLTE` package.

```sh
build/scss/_control-sidebar.scss:    right: -$control-sidebar-width;
build/scss/_control-sidebar.scss:    width: $control-sidebar-width;
build/scss/_control-sidebar.scss:      margin-right: $control-sidebar-width;
build/scss/_control-sidebar.scss:      margin-right: $control-sidebar-width;
build/scss/_sidebar-mini.scss:        width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 1rem);
build/scss/_sidebar-mini.scss:          width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 2rem);
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 3rem);
build/scss/_sidebar-mini.scss:              width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 4rem);
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 5rem);
build/scss/_sidebar-mini.scss:        width: $sidebar-width;
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width} - 1rem);
build/scss/_sidebar-mini.scss:              width: calc(#{$sidebar-width} - 1rem - 1rem);
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - 1rem - 2rem);
build/scss/_sidebar-mini.scss:                  width: calc(#{$sidebar-width} - 1rem - 3rem);
build/scss/_sidebar-mini.scss:                    width: calc(#{$sidebar-width} - 1rem - 4rem);
build/scss/_sidebar-mini.scss:        width: $sidebar-width;
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width});
build/scss/_sidebar-mini.scss:              width: calc(#{$sidebar-width} - .2rem);
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - .2rem * 2);
build/scss/_sidebar-mini.scss:                  width: calc(#{$sidebar-width} - .2rem * 3);
build/scss/_sidebar-mini.scss:                    width: calc(#{$sidebar-width} - .2rem * 4);
build/scss/_sidebar-mini.scss:        width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - .5rem);
build/scss/_sidebar-mini.scss:          width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 1rem);
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 1.5rem);
build/scss/_sidebar-mini.scss:              width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 2rem);
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2 - 2.5rem);
build/scss/_sidebar-mini.scss:        width: $sidebar-width;
build/scss/_sidebar-mini.scss:          width: calc(#{$sidebar-width} - .5rem);
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width} - .5rem * 2);
build/scss/_sidebar-mini.scss:              width: calc(#{$sidebar-width} - .5rem * 3);
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - .5rem * 4);
build/scss/_sidebar-mini.scss:                  width: calc(#{$sidebar-width} - .5rem * 5);
build/scss/_sidebar-mini.scss:      width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2);
build/scss/_sidebar-mini.scss:          width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2);
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2);
build/scss/_sidebar-mini.scss:            width: $sidebar-width;
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - 1rem);
build/scss/_sidebar-mini.scss:                  width: calc(#{$sidebar-width} - 1rem - 1rem);
build/scss/_sidebar-mini.scss:                    width: calc(#{$sidebar-width} - 1rem - 2rem);
build/scss/_sidebar-mini.scss:                      width: calc(#{$sidebar-width} - 1rem - 3rem);
build/scss/_sidebar-mini.scss:                        width: calc(#{$sidebar-width} - 1rem - 4rem);
build/scss/_sidebar-mini.scss:            width: $sidebar-width;
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width});
build/scss/_sidebar-mini.scss:                  width: calc(#{$sidebar-width} - .2rem);
build/scss/_sidebar-mini.scss:                    width: calc(#{$sidebar-width} - .2rem * 2);
build/scss/_sidebar-mini.scss:                      width: calc(#{$sidebar-width} - .2rem * 3);
build/scss/_sidebar-mini.scss:                        width: calc(#{$sidebar-width} - .2rem * 4);
build/scss/_sidebar-mini.scss:            width: calc(#{$sidebar-width} - #{$sidebar-padding-x} * 2);
build/scss/_sidebar-mini.scss:            width: $sidebar-width;
build/scss/_sidebar-mini.scss:              width: calc(#{$sidebar-width} - .5rem);
build/scss/_sidebar-mini.scss:                width: calc(#{$sidebar-width} - .5rem * 2);
build/scss/_sidebar-mini.scss:                  width: calc(#{$sidebar-width} - .5rem * 3);
build/scss/_sidebar-mini.scss:                    width: calc(#{$sidebar-width} - .5rem * 4);
build/scss/_sidebar-mini.scss:                      width: calc(#{$sidebar-width} - .5rem * 5);
build/scss/_variables.scss:$sidebar-width: 250px !default;
build/scss/_variables.scss:$control-sidebar-width: $sidebar-width !default;
build/scss/mixins/_sidebar.scss:        width: $sidebar-width;
build/scss/mixins/_sidebar.scss:          width: $sidebar-width;
build/scss/mixins/_sidebar.scss:          width: $sidebar-width;
build/scss/_layout.scss:        width: $sidebar-width;
build/scss/_layout.scss:      width: $sidebar-width;
build/scss/_layout.scss:              width: $sidebar-width;
build/scss/_layout.scss:          width: $sidebar-width;
build/scss/_layout.scss:      margin-left: $sidebar-width;
build/scss/_layout.scss:      margin-left: $sidebar-width;
build/scss/_layout.scss:      margin-left: $sidebar-width;
build/scss/_layout.scss:    width: $sidebar-width;
build/scss/_layout.scss:      margin-left: -$sidebar-width;
build/scss/_layout.scss:      margin-left: -$sidebar-width;
build/scss/_layout.scss:    width: $sidebar-width;
```

So, I believe these changes will introduce compatibility problems with the underlying `AdminLTE` package. This type of configuration should be available on the `AdminLTE` package, for example, using `data-sidebar-width` attribute on the `body` tag. Maybe they are planning something like this for new versions of the package ( @REJack ).

To summarize, I do not consider these changes are good here. However I will give a try when I get some time to see if we can use this as a workaround until this configuration is available on the underlying package.
